### DISCUSSION TITLE
Connecting to excel/files

### DISCUSSION THREAD

**Discussion by writetokarthik** [apache/superset#17250] [General]
Hello, I am new to superset and exploring for using superset for dashboarding needs. Would like to know if superset can connect to MS Excel/Google sheets as source for data. 

Also looking for any recommendation on ways to get started with superset. 
 - karthik

**Comment by kamalkeshavani-aiinside** [comment]
Superset latest released version 1.3.2 supports connecting to Public Google sheets as data source.
Here is how you can connect: https://docs.preset.io/docs/google-sheets-connection

In next release, private google sheets will also be supported to connect.
For Excel files, you will need a database where you can upload your csv/excel file as a table, then use it as your data source. But this way you need to always update your table with latest data from excel file.

**Comment by duniang818** [comment]
this way i can import csv file, but dont import excel file.
the error: excel file format cannot be determined, you must specify an engine manually.

**Comment by sfirke** [comment]
Closing as resolved, since the answer above is correct: Superset supports CSV/Excel uploading to the data warehouse as well as connecting to Google Sheets.
